### PR TITLE
Add vtk.util.colors for backward compability

### PR DESCRIPTION
**Pull request by skoudoro**
Some examples do not work on `Dipy` because `fury.window.colors` do not exist.

The aim of this PR is to fix this small issue.

I use this opportunity to:
- Update readme
- Add codecov.yml
- Add coveragerc
- Add Issue template

### CODE REVIEW

File: ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+## Description
+[Please provide a general introduction to the issue/proposal.]
+
+[If reporting a bug, attach the entire traceback from Python and follow the way to reproduce below]
```
Comment (dmreagan) [inline ISSUE_TEMPLATE.md]:
```suggestion
[If reporting a bug, attach the entire traceback from Python and follow the way to reproduce below.]
```

File: README.rst [hunk 2]
```diff
@@ -12,12 +8,62 @@ FURY
         :target: https://codecov.io/gh/fury-gl/fury
 
 
-Free Unified Rendering in Python
+=======================================
+FURY - Free Unified Rendering in Python
+=======================================
+
+
+FURY is package for scientific visualization with Python.
```
Comment (dmreagan) [inline README.rst]:
```suggestion
FURY is a package for scientific visualization with Python.
```

File: README.rst [hunk 3]
```diff
@@ -12,12 +8,62 @@ FURY
...
+
+from binaries
+~~~~~~~~~~~~~
+
+    pip install fury
+
+from source
+~~~~~~~~~~~
+
+**Step 1.** Get latest source by cloning this repo::
```
Comment (dmreagan) [inline README.rst]:
```suggestion
**Step 1.** Get the latest source by cloning this repo::
```

File: README.rst [hunk 4]
```diff
@@ -12,12 +8,62 @@ FURY
...
+**Step 4:** Enjoy!
+
+For more information, see also `installation page on fury.gl <https://fury.gl/stable/installation.html>`_
+
+Contribute
+----------
+
+We love contributions!
+
+Whatever your experience level, every new contributors are welcomed!
```
Comment (dmreagan) [inline README.rst]:
```suggestion
Whatever your experience level, all new contributors are welcomed!
```

File: README.rst [hunk 5]
```diff
@@ -12,12 +8,62 @@ FURY
...
+
+You've discovered a bug or something else you want to change - excellent! Create an `issue <https://github.com/fury-gl/fury/issues/new>`_!
 
-* Free software: 3-clause BSD license
-* Documentation: https://fury.gl
+You've worked out a way to fix it – even better! submit a Pull Request!
```
Comment (dmreagan) [inline README.rst]:
```suggestion
You've worked out a way to fix it – even better! Submit a Pull Request!
```

File: README.rst [hunk 6]
```diff
@@ -12,12 +8,62 @@ FURY
...
 
-Features
---------
+You want to tell us about it – best of all!
 
-* TODO
+Start at the `contributing guide <CONTRIBUTING.rst>`_!
```
Comment (dmreagan) [inline README.rst]:
```suggestion
Start with the `contributing guide <CONTRIBUTING.rst>`_!
```

File: README.rst [hunk 7]
```diff
@@ -12,12 +8,65 @@ FURY
         :target: https://codecov.io/gh/fury-gl/fury
 
 
-Free Unified Rendering in Python
+FURY - Free Unified Rendering in Python
+=======================================
+
+
+FURY is a package for scientific visualization with Python.
```
Comment (Garyfallidis) [inline README.rst]: FURY is a software library for scientific visualization in Python

File: README.rst [hunk 8]
```diff
@@ -12,12 +8,65 @@ FURY
...
+
+For more information, see also `installation page on fury.gl <https://fury.gl/stable/installation.html>`_
+
+Contribute
+==========
+
+
+We love contributions!
+
+Whatever your experience level, all new contributors are welcomed!
```
Comment (Garyfallidis) [inline README.rst]: Remove this entire sentence. Just add something saying:
Start your pull request here (and here add the link to the pull requests of fury).

File: README.rst [hunk 9]
```diff
@@ -12,12 +8,65 @@ FURY
...
+
+You've discovered a bug or something else you want to change - excellent! Create an `issue <https://github.com/fury-gl/fury/issues/new>`_!
 
-* Free software: 3-clause BSD license
-* Documentation: https://fury.gl
+You've worked out a way to fix it – even better! Submit a Pull Request!
 
-Features
---------
+You want to tell us about it – best of all!
```
Comment (Garyfallidis) [inline README.rst]: best of all?

**Comment by Garyfallidis** [comment]
Thanks for the update. Happy releasing!

**Pull request merged**